app/mailers/event/published_mailer.rb

```diff
@@ -14,8 +14,10 @@ def notice(course)
     @course = course
     headers = {bcc: course.groups.first.course_admin_email}
     locales = course.language.split("_")
+    event_leaders = course.participations.joins(:roles, :person)
+      .where(roles: {type: Event::Role::Leader.sti_name}).map(&:person)
```

Comment:
```suggestion
   RELEVANT_EVENT_ROLE_TYPES = [Event::Role::Leader, Event::Role::AssistantLeader]

    event_leaders = Person.where(id: course.participations.joins(:roles).where(roles: {type: RELEVANT_EVENT_ROLE_TYPES.map(&:sti_name)}).pluck(:person_id)
```
Hier sind beide Leitungsrollen betroffen